libsql-server/src/connection/libsql.rs

```diff
@@ -76,60 +60,62 @@ where
     ) -> Result<Self> {
         let txn_timeout = config_store.get().txn_timeout.unwrap_or(TXN_TIMEOUT);
 
-        let mut this = Self {
-            db_path,
-            stats,
-            broadcaster,
-            config_store,
-            extensions,
-            max_response_size,
-            max_total_response_size,
-            auto_checkpoint,
-            current_frame_no_receiver,
-            _db: None,
-            wal_wrapper,
-            encryption_config,
-            block_writes,
-            resolve_attach_path,
-            connection_manager: ConnectionManager::new(txn_timeout),
-            make_wal_manager,
-        };
-
-        let db = this.try_create_db().await?;
-        this._db = Some(db);
-
-        Ok(this)
-    }
+        let (snd, recv) = async_channel::bounded(16);
```

Comment:
Noob rust question: `snd` ownership will be transferred to the tokio coroutine or how does this work here?

Reply:
That's correct, it's moved in the async block